src/components/MarkdownProvider/components/CodeExample/index.tsx

```diff
@@ -119,7 +119,7 @@ export const CodeExample: React.FC<ICodeExampleProps> = ({ children, code }) =>
             <DirectionRtlStroke />
           </ToggleIconButton>
         </Tooltip>
-        <ColorPickerDialog color={color} onChange={setColor}>
+        <ColorPickerDialog color={color} onChange={setColor} placement="bottom-end">
```

Comment:
Improves the `ColorPickerDialog` placement on narrow screens

![Screenshot 2024-08-01 at 10 04 50 AM](https://github.com/user-attachments/assets/8c5ec04d-060f-4e6d-9836-7256b02ea77c)

Reply:
Interesting – I would have expected TooltipModal to automatically reposition the dialog here since it's value is defaulting to `auto`. 

https://github.com/zendeskgarden/react-components/blob/next/packages/modals/src/elements/TooltipModal/TooltipModal.tsx#L182

Was that not the case here?

Reply:
It's set to `bottom-start` by default. In our case, that doesn't work well because of the picker location.

https://github.com/zendeskgarden/react-components/blob/3da9488a98d57c4150b0308aa67b9f561c4ef929/packages/colorpickers/src/elements/ColorSwatchDialog/index.tsx#L200-L204

Reply:
That looks like the color picker dialog in the screen shot, is that what we're talking about? (You linked to color swatch dialog).

Reply:
Oh I see now - both use bottom-start by default. 😅

In that case it looks good. I guess I was wondering if "auto" would achieve the desired result and be less specific to a single position.

Reply:
Ooops, my bad.😓  That's what I meant. Both have the same `bottom-start` default.